components/suggestion/switch_channel_provider.jsx

```diff
@@ -283,7 +283,7 @@ export default class SwitchChannelProvider extends Provider {
             // Fetch data from the server and dispatch
             this.fetchUsersAndChannels(channelPrefix, resultsCallback);
         } else {
-            this.formatUnreadChannelsAndDispatch(resultsCallback);
```

Comment:
should `formatUnreadChannelsAndDispatch` function be removed if it's not being used?

Reply:
I think I removed this one from the code.